### ISSUE TITLE
RandomForest.mtry initialization

### ISSUE THREAD

**Issue by takanori-ugai** [haifengl/smile#420]
Validation.cv(10, RandomForest.Trainer(df.attributes(),(3)),ds.x(), ds.labels())
should work.
But I get
java.lang.IllegalArgumentException: Invalid number of variables to split on at a node of the tree: -1
	at smile.classification.RandomForest.<init>(RandomForest.java:610)
	at smile.classification.RandomForest$Trainer.train(RandomForest.java:262)
	at smile.classification.RandomForest$Trainer.train(RandomForest.java:120)
	at smile.validation.Validation.cv(Validation.java:356)

Validation.cv(10, RandomForest.Trainer(df.attributes(),(3)).setNumRandomFeatures(2),df.x(), df.labels()) is work.

RandomForest.mtry must be initialized when a RnadomForest class instance is generated.

PR is #419

**Comment by haifengl** [comment]
Thanks

**Comment by takanori-ugai** [comment]
PR #419 is merged.
Thanks.
